ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ValetTouchIDTestViewController.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+final class ValetTouchIDTestViewController : UIViewController
+{
+    // MARK: Properties
+
+    @IBOutlet var textView : UITextView?
+    let secureEnclaveValet : VALSecureEnclaveValet = VALSecureEnclaveValet.init(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence) as VALSecureEnclaveValet!
```

Comment:
The major issue is that the initializer is nullable so it's returning a `VALSecureEnclaveValet?` What I'm doing is probably poor form, though, I'll think about it a bit more